copier.yml

```diff
@@ -1,14 +1,7 @@
 # Configure jinja2 defaults to make syntax highlighters happy
 _templates_suffix: .jinja
 _subdirectory: src
-_envops:
-  block_end_string: "%}"
-  block_start_string: "{%"
-  comment_end_string: "#}"
-  comment_start_string: "{#"
-  keep_trailing_newline: true
-  variable_end_string: "}}"
-  variable_start_string: "{{"
```

Comment:
@yajo does this mean the default jinja syntax changed with copier 9?

Reply:
It changed many versions ago. We just didn't have a constraint, so we had to define it manually.